client/src/components/wellness/SupportOthers.tsx

```diff
@@ -0,0 +1,30 @@
+import { ReactNode } from 'react';
+
+export default function SupportOthers() {
+    return (
+        <div className="space-y-4">
+            <h2 className="text-xl font-semibold">Support Others</h2>
+            <div className="space-y-3">
+                <div className="bg-red-50 dark:bg-red-900/20 border border-red-200 dark:border-red-700 rounded p-3">
+                    <h3 className="font-medium text-red-800 dark:text-red-200">Emergency Services</h3>
+                    <p className="text-red-700 dark:text-red-300 font-bold text-lg">911</p>
+                    <p className="text-sm text-red-600 dark:text-red-400">Request a crisis intervention team</p>
+                </div>
+            </div>
+            If you are concerned for a friend, classmate, or another student and would like Wellness to check in with them, please email <a href="mailto:gunnwellnessteam@pausd.org">gunnwellnessteam@pausd.org</a> with:
```

Comment:
encapsulate w/ paragraph for proper spacing

```suggestion
            <p>If you are concerned for a friend, classmate, or another student and would like Wellness to check in with them, please email <a href="mailto:gunnwellnessteam@pausd.org">gunnwellnessteam@pausd.org</a> with:</p>
```